src/lightmap.cpp

```diff
@@ -273,6 +273,17 @@ void map::generate_lightmap()
                 apply_light_source(mx, my, critter.type->luminance, trigdist);
             }
         }
+        if (INBOUNDS(mx, my)) {
```

Comment:
This is already tested just some lines above, please move the check for the glowing effect into the block above.